sentry_sdk/integrations/asyncio.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import
+
+import asyncio
```

Comment:
I would wrap the asyncio import in a `try..except ImportError` like all other integrations

Reply:
done.